airbyte-integrations/connectors/source-jdbc/src/main/java/io/airbyte/integrations/source/jdbc/AbstractJdbcSource.java

```diff
@@ -268,23 +269,15 @@ private JsonNode getColumnMetadata(final ResultSet resultSet) throws SQLExceptio
         .build());
   }
 
-  /**
```

Comment:
Wow! I'm surprised this method even existed given how trivial this appears. Great job cleaning this up